src/Raven.Studio/typescript/common/appUrl.ts

```diff
@@ -596,10 +597,11 @@ class appUrl {
         return "#databases/tasks/editReplicationSinkTask?" + databasePart + taskPart;
     }
 
-    static forEditPeriodicBackupTask(db: database, taskId?: number): string {
+    static forEditPeriodicBackupTask(db: database, sourceView: EditPeriodicBackupTaskSourceView,taskId?: number): string {
```

Comment:
let's handle case when sourceView parameter doesn't exist.

Reply:
Here I made this required on purpose. If we don't specify it somewhere in the code, there will be an typesctript error. However, if the url is missing sourceView, it redirects to Backups by default. 

https://github.com/ravendb/ravendb/blob/02890f9afc7a4f051ae646672fdd27b022d6d386/src/Raven.Studio/typescript/viewmodels/database/tasks/editPeriodicBackupTask.ts#L228-L239